### PR TITLE
Add support for unfollow confirmation

### PR DESCRIPTION
**Pull request by sionking**
Please verify the logic it is late night now :)

### CODE REVIEW

**Comment by sionking** [comment]
Xpath not working please assist

File: instapy/unfollow_util.py [hunk 1]
```diff
@@ -143,29 +143,49 @@ def unfollow(browser,
...
                     if following:
                         # click the button
                         click_element(browser, follow_button) # follow_button.click()
                         sleep(4)
 
-                        # double check not following
-                        follow_button = browser.find_element_by_xpath(
-                            "//*[contains(text(), 'Follow')]")
-
+                        #
```
Comment (CharlesCCC) [inline instapy/unfollow_util.py:152]: empty comment ?

File: instapy/unfollow_util.py [hunk 2]
```diff
@@ -143,29 +143,49 @@ def unfollow(browser,
...
+                                # not having confirmation
+                                # unfollowed = False
+                                pass
+
+
 
-                            unfollowNum += 1
-                            update_activity('unfollows')
 
+                        sleep(9)
```
Comment (CharlesCCC) [inline instapy/unfollow_util.py:171]: why we need sleep 9 sec here ?

File: instapy/unfollow_util.py [hunk 3]
```diff
@@ -143,29 +143,49 @@ def unfollow(browser,
...
+                            # save any unfollowed person
+                            log_record_all_unfollowed(username, person, logger, logfolder)
+
                             logger.info(
                                 '--> Ongoing Unfollow From InstaPy {},'
                                 ' now unfollowing: {}'
                                 .format(str(unfollowNum), person.encode('utf-8')))
 
-                            sleep(15)
+                            sleep(8)
```
Comment (CharlesCCC) [inline instapy/unfollow_util.py:185]: same here, how did we decide when/how long should each step to sleep ? it sounds like we just pull the number out of the air ?

**Comment by CharlesCCC** [comment]
@sionking  thank you for the quick PR, I did a quick review and looks fine for the most part, got some small questions for you.

File: instapy/unfollow_util.py (hunk 3 above)
Comment (sionking) [reply instapy/unfollow_util.py:185]: Works well for my clients, sleep should be 20~ each unfollow

File: instapy/unfollow_util.py (hunk 1 above)
Comment (sionking) [reply instapy/unfollow_util.py:152]: late night PR

**Comment by sionking** [comment]
@CharlesCCC Thanks!

**Comment by sionking** [comment]
I see there is a PR that fit better. this one closed.

**Comment by uluQulu** [comment]
Those changes looks neat, @sionking!
Why close?

**Comment by timgrossmann** [comment]
@uluQulu You're right, those are actually some really nice adjustments.
I think we should try to still merge them.

**Pull request closed without merging**